tutorials/monitoring-demo/walkthrough-setup.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Monitoring NServiceBus Demo - Setup instructions"
+reviewed: 2018-03-15
+summary: Detailed instructions for setting up the NServiceBus monitoring demo
+---
+
+This document explains how to set up the [NServiceBus monitoring demo])(./) in detail.
```

Comment:
Is this the right link?

Reply:
I used the same link as in here: https://github.com/Particular/docs.particular.net/edit/master/tutorials/monitoring-demo/walkthrough-solution.include.md